plugins/jobtype/jobtypes/teradataToHdfs/private.properties

```diff
@@ -0,0 +1,8 @@
+jobtype.class=azkaban.jobtype.connectors.TeradataToHdfsJob
+jobtype.classpath=lib/*
+
+jobtype.lib.dir=${plugin.dir}/lib
+jobtype.tdwallet.jar=${jobtype.lib.dir}/teradata-connector-1.4.1.jar
+
+libjars.home=${plugin.dir}/libjars
+libjars=${jobtype.lib.dir}/teradata-connector-1.4.1.jar,${libjars.home}/avro-1.7.7.jar,${libjars.home}/terajdbc4.jar,${libjars.home}/tdgssconfig.jar,${libjars.home}/avro-mapred-1.7.7-hadoop2.jar,${libjars.home}/paranamer-2.6.jar,${libjars.home}/opencsv-2.3.jar"
```

Comment:
We have observed failures when we specified individual files and someone upgraded versions (filename). If we are using all the jars in libjars directory, then we can list all the expected files in comment and just use 
libjars = ${jobtype.lib.dir}/* ?